demo-shell-ng2/app/components/files/files.component.html

```diff
@@ -268,10 +268,10 @@
         </md-input-container>
     </section>
     <div *ngIf="!acceptedFilesTypeShow">
-        <alfresco-upload-button
+        <adf-upload-button
             #uploadButton
             tooltip="Custom tooltip"
-            [disabled]="!enableUpload"
+            [disable]="!enableUpload"
```

Comment:
'disabled' is a well-known attribute name for all HTML elements. why do we need renaming it to "disable" instead?

Reply:
Example: https://www.w3schools.com/tags/att_input_disabled.asp

Reply:
because it's not a valid input attribute name, 'cause in this way the value in input won't change. But due to compliance of name we have found a better solution @Input('disabled').